src/routes/+layout.svelte

```diff
@@ -15,6 +15,7 @@
 	initializeStores();
 	const drawerStore = getDrawerStore();
 
+	$: isDrawerOpen = $drawerStore.open ? true : false;
```

Comment:
this is smart and shows that you know your way around svelte! But it is just hiding 2 mistakes that I made. It is a hotfix. We shouldn't use it if we can find the actual mistake that I made...